### ISSUE TITLE
Error: Could not establish connection. Receiving end does not exist.

### ISSUE THREAD

**Issue by ohhheyyyy** [dequelabs/axe-core#2580]
Expectation: While logged in with correct Deque Systems credentials, click the analyze button in the axe section of Chrome DevTools; see report of accessibility violations on page.

Actual: While logged in with correct Deque Systems credentials, click the analyze button the axe section of Chrome DevTools; unable to see report of accessibility violations on page and instead see error:

`An error occurred`

---
Please try again. If the error persists, please [report the issue ](https://github.com/dequelabs/axe-core/issues)to Deque Systems 

```
Error details ˅
Error: Could not establish connection. Receiving end does not exist.
```

Motivation: Cannot use to evaluate accessibility violations and warnings.

<pre><code>
axe-core version: 4.6.1
</code></pre>

### Screen shot below ⬇️

![axeBeta](https://user-images.githubusercontent.com/5541812/96766156-a9416200-13a8-11eb-95b2-697f69db2c3b.png)

**Comment by scurker** [comment]
@ohhheyyyy Is this something that occurs against any URL, or it specific URLs?

**Comment by ohhheyyyy** [comment]
Hey @scurker! This happens on any website I've tried – localhost, site behind VPN, publicly accessible sites like news orgs, etc.

**Comment by scurker** [comment]
In your extension settings (chrome://extensions/?id=lhdoppojpmngadmnindnejefpokejbdd), what does your _Site Access_ look like?

![chrome extension site access settings with on click, on specific sites, and all sites radio options](https://user-images.githubusercontent.com/1062039/97187957-fe96be00-1770-11eb-82bb-d7edddf73d93.png)

I believe by default chrome has _On Click_ selected, which means you would need to grant access each time by clicking on the Axe icon in your chrome menu bar.

**Comment by ohhheyyyy** [comment]
This was precisely the issue @scurker. Thank you for your replies and specifics to resolve. Greatly appreciate your work!!

**Comment by scurker** [comment]
Glad you got it figured out! 🎉
